Letterbook.Core/CoreOptions.cs

```diff
@@ -8,4 +8,15 @@ public class CoreOptions
 	public string Scheme { get; set; } = "https";
 	public string Port { get; set; } = "443";
 	public int MaxCustomFields { get; set; } = 10;
+
+	public Uri GetInstanceUri()
```

Comment:
There's an extension method that does a very similar thing. Would it work to use that instead?

I wanted to keep Options classes as just data containers if we can.

Reply:
It would, yes!